otterdog/adoptium.jsonnet

```diff
@@ -655,6 +655,23 @@ orgs.newOrg('adoptium') {
         orgs.newBranchProtectionRule('master'),
       ],
     },
+    orgs.newRepo('jdk22') {
+      allow_update_branch: false,
+      default_branch: "master",
+      dependabot_security_updates_enabled: true,
+      secret_scanning: "disabled",
+      secret_scanning_push_protection: "disabled",
+      web_commit_signoff_required: false,
+    },
+    orgs.newRepo('temurin23-binaries') {
```

Comment:
should this be temurin23-binaries or rather temurin22-binaries to be inline with the repo above?